cloudnet/src/main/java/de/dytanic/cloudnet/network/NetworkClientChannelHandlerImpl.java

```diff
@@ -26,6 +26,7 @@ public void handleChannelInitialize(INetworkChannel channel) {
             return;
         }
 
+        System.out.println(connectionWhichSendRequest.get());
```

Comment:
Is this a debug output? You might want to remove this

Reply:
removed